x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -14,3 +14,4 @@ export const LIMITED_CONCURRENCY_ENDPOINT_COUNT = 100;
 
 export const TRUSTED_APPS_LIST_API = '/api/endpoint/trusted_apps';
 export const TRUSTED_APPS_CREATE_API = '/api/endpoint/trusted_apps';
+export const TRUSTED_APPS_DELETE_API = `${TRUSTED_APPS_LIST_API}/{id}`;
```

Comment:
nit: maybe follow the pattern of being manual. had to do a double take and read char-by-char to see if create API differed from the list API, as doing this implies.

Or change create definition to follow the var way

Reply:
I see. I will spell it out 👍